ch00python/010exemplar.ipynb

```diff
@@ -267,37 +247,33 @@
    "metadata": {},
    "source": [
     "We'll fetch a map of a place from the Google Maps server, given a longitude and latitude.\n",
-    "The URLs look like: https://mt0.google.com/vt?x=658&y=340&z=10&lyrs=s.    \n",
-    "The latitude & longitude need to be translated to the coordinate system used by Google Maps and we'll frequently be generating these URLs.\n",
-    "So we'll make ourselves another function to build up a request from the URL given our parameters."
+    "The URLs look like: `https://mt0.google.com/vt?x=658&y=340&z=10&lyrs=s`.  \n",
+    "Since we'll frequently be generating these URLs, we will create two helper functions to make our life easier.\n",
+    "The first is a function to convert our latitude & longitude into the coordinate system used by Google Maps. You can find details of the mathematics can be [here](https://wiki.openstreetmap.org/wiki/Slippy_map_tilenames#X_and_Y).\n",
```

Comment:
```suggestion
    "The first is a function to convert our latitude & longitude into the coordinate system used by Google Maps. You can find details of the mathematics [here](https://wiki.openstreetmap.org/wiki/Slippy_map_tilenames#X_and_Y).\n",
```

We try to avoid links on words like "here" as it doesn't help with screen readers -- although the whole page may not hugely accessible anyway, so perhaps a moot point. Could do something more like:
```suggestion
    "The first is a function to convert our latitude & longitude into the coordinate system used by Google Maps. If interested, you can see [more details of the mathematics](https://wiki.openstreetmap.org/wiki/Slippy_map_tilenames#X_and_Y).\n",
```